pyproject.toml

```diff
@@ -46,6 +46,7 @@ dependencies = [
 dynamic = ['version']
 
 [project.optional-dependencies]
+test = ["pytest>=8.3.0", "pytest-mock>=3.14.0", "pytest-examples>=0.0.12"]
```

Comment:
I think we don't need this change

Reply:
Could I add these in another pull request? I think it will make it easier to get things up and running for other contributors.

Reply:
probably need to improve the readme and include a contribution doc.
we have some useful commands in [Makefile](https://github.com/pydantic/pydantic-settings/blob/main/Makefile) that makes development easy like:

- `make install`
- `make format`
- `make lint`
- ...

Reply:
Oh okay! Funny enough I looked for one earlier. I'll revert this change.